File: Day-01-Bit-Manipulation/08-XOR-Queries/solution.py
n = int(input())
arr = list(map(int, input().split()))

q = int(input())
for _ in range(q):
    l, r = map(int, input().split())
    res = 0
    for i in range(l, r+1):
        res ^= arr[i]
    print(res, end=" ")

File: Day-01-Bit-Manipulation/08-XOR-Queries/solution.cpp
#include <iostream>
using namespace std;
int main() {
    int n, q;
    cin >> n;

    int arr[n];
    for(int i = 0; i < n; i++)
        cin >> arr[i];

    cin >> q;
    while(q--) {
        int l, r, res = 0;
        cin >> l >> r;

        for(int i = l; i <= r; i++)
            res ^= arr[i];

        cout << res << " ";
    }
}